.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this vscode directory required to be on the repo ? Better to remove IDE specific stuff. @scamper07

Reply:
Removed vscode folder from git